kubernetes/oci-cloud-controller/user.tf

```diff
@@ -0,0 +1,46 @@
+resource "tls_private_key" "cloud_controller_user_key" {
+  count = "${var.cloud_controller_user_ocid==""? 1 : 0 }"
+  algorithm   = "RSA"
+  rsa_bits = 2048
+}
+
+resource "oci_identity_group" "cloud_controller_group" {
+  count = "${var.cloud_controller_user_ocid==""? 1 : 0 }"
+  name = "${var.label_prefix}cloud_controller_group"
+  description = "Terraform created group for OCI Cloud controller manager"
```

Comment:
Consistent capitalisation 

Terraform created group for OCI Cloud Controller manager